sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/LiveMetrics/DataCollection/DocumentHelper.cs

```diff
@@ -321,5 +375,17 @@ internal static bool IsHttpSuccess(Activity activity, string? responseCode)
                 return activity.Status != ActivityStatusCode.Error;
             }
         }
+
+        private static void SetProperties(DocumentIngress documentIngress, ActivityTagsProcessor atp)
```

Comment:
same comment as above, is this a requirement?

Reply:
This is used for Dependencies which will have several extra attributes that don't add value to Properties.
In the case of Dependencies, we already have an instance of `ActivityTagsProcessor` so it's no extra effort to read the `UnMappedTags`. 

We do have a TODO to find an alternative to ActivityTagsProcessor.